src/components/FindNamedEntitiesForm.vue

```diff
@@ -78,20 +86,36 @@ export default {
   mixins: [utils],
   props: {
     /**
-     * Callback function to call when the form have been submitted (this should be replaced by an event in future versions).
+     * Dark mode background option
+     */
+    dark: {
+      type: Boolean
+    },
+    /**
+     * Project name to select in the input instead of default project
      */
-    finally: {
-      type: Function,
-      default: noop
+    projectName: {
+      type: String,
+      default: null
+    },
+    /**
+     * Disable project selection select input
+     */
+    disableProjectSelection: {
```

Comment:
It's good you block user from changing the project but I would completely hide the project input instead. No need to show it as it's obvious which project is being selected.

Reply:
ok I'm changing it to hide the input